src/cascadia/TerminalApp/Pane.h

```diff
@@ -73,7 +73,7 @@ class Pane : public std::enable_shared_from_this<Pane>
 
     std::shared_ptr<Pane> GetActivePane();
     winrt::Microsoft::Terminal::Control::TermControl GetLastFocusedTerminalControl();
-    winrt::Microsoft::Terminal::Control::TermControl GetTerminalControl();
+    winrt::Microsoft::Terminal::Control::TermControl GetTerminalControl() const;

```

Comment:
There's a few of these like `GetTerminalControl()` and `GetProfile()` that kinda don't make sense here anymore. Why not do this:
- move them to `TerminalPaneContent`
- force callers to `GetContent().GetTerminalControl()`
...or something like that? That way it feels like a cleaner abstraction/separation between the Pane and underlying content.

Reply:
You know what happened here - the remaining callers got merged into main mostly _after_ I had written most of this. The broadcast input stuff? All merged after I did most of this initially 🤦

Reply:
I wanna say that this is probably okay, mainly for the reason that when I went to go solve the last few cases here (mainly broadcasting related, and also one for detaching termcontrol's from a window), I ended up with a bunch of 

```c++
TermControl& _termControlFromPane(const auto& pane)
{
    if (const auto content{ pane->GetContent() })
    {
        if (const auto termContent{ content.try_as<winrt::TerminalApp::TerminalPaneContent>() })
        {
            return termContent.GetTerminal();
        }
    }
    return nullptr;
}
```

helpers scattered through the code. Almost seemed more sensible to just leave it in `Pane`. Maybe rename it to `MaybeGetTerminalControl`? idk